client/my-sites/site-settings/taxonomies/index.jsx

```diff
@@ -13,14 +14,16 @@ import HeaderCake from 'components/header-cake';
 import TaxonomyManager from 'blocks/taxonomy-manager';
 import { getSelectedSite, getSelectedSiteId } from 'state/ui/selectors';
 import { getPostTypeTaxonomy } from 'state/post-types/taxonomies/selectors';
+import DocumentHead from 'components/data/document-head';
 
 const Taxonomies = ( { labels, postType, site, taxonomy } ) => {
 	const goBack = () => {
 		page( '/settings/writing/' + site.slug );
 	};
 
 	return (
-		<div className="main main-column" role="main">
+		<div className="main main-col	umn" role="main">
+			<DocumentHead title={ i18n.translate( 'Manage %(taxonomy)s', { args: { taxonomy: labels.name } } ) } />
```

Comment:
I think the current "practice" to i18n is using the `localize` Higher order Component. https://github.com/Automattic/i18n-calypso#usage-2